tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -442,44 +442,44 @@ def validate(instance):
     def validate(instance):
-        FAMILY_AFFILIATION = (
-            instance["FAMILY_AFFILIATION"]
-            if type(instance) is dict
-            else getattr(instance, "FAMILY_AFFILIATION")
-        )
-        RELATIONSHIP_HOH = (
-            instance["RELATIONSHIP_HOH"]
-            if type(instance) is dict
-            else getattr(instance, "RELATIONSHIP_HOH")
-        )
-        RELATIONSHIP_HOH = int(RELATIONSHIP_HOH)
-        MONTHS_FED_TIME_LIMIT = (
-            instance["MONTHS_FED_TIME_LIMIT"]
-            if type(instance) is dict
-            else getattr(instance, "MONTHS_FED_TIME_LIMIT")
-        )
-        if FAMILY_AFFILIATION == 1 and (RELATIONSHIP_HOH == 1 or RELATIONSHIP_HOH == 2):
-            if MONTHS_FED_TIME_LIMIT is None or int(MONTHS_FED_TIME_LIMIT) < 1:
-                return (False,
-                        "If FAMILY_AFFILIATION == 2 and MONTHS_FED_TIME_LIMIT== 1 or 2,"
-                        + " then MONTHS_FED_TIME_LIMIT > 1.",
-                        ['FAMILY_AFFILIATION', 'MONTHS_FED_TIME_LIMIT']
-                        )
-            else:
-                return (
-                    True,
-                    None,
-                    ["FAMILY_AFFILIATION", "RELATIONSHIP_HOH", "MONTHS_FED_TIME_LIMIT"],
-                )
-        else:
-            return (
-                True,
-                None,
-                ["FAMILY_AFFILIATION", "RELATIONSHIP_HOH", "MONTHS_FED_TIME_LIMIT"],
+        false_case = (False,
+                      "If FAMILY_AFFILIATION == 2 and COUNTABLE_MONTH_FED_TIME== 1 or 2, then "
+                      + "COUNTABLE_MONTH_FED_TIME > 1.",
+                      ["FAMILY_AFFILIATION", "RELATIONSHIP_HOH", "COUNTABLE_MONTH_FED_TIME",],
+                      )
+        true_case = (True,
+                     None,
+                     ["FAMILY_AFFILIATION", "RELATIONSHIP_HOH", "COUNTABLE_MONTH_FED_TIME",],
+                     )
+        try:
+            FAMILY_AFFILIATION = (
+                instance["FAMILY_AFFILIATION"]
+                if type(instance) is dict
+                else getattr(instance, "FAMILY_AFFILIATION")
             )
+            RELATIONSHIP_HOH = (
+                instance["RELATIONSHIP_HOH"]
+                if type(instance) is dict
+                else getattr(instance, "RELATIONSHIP_HOH")
+            )
+            RELATIONSHIP_HOH = int(RELATIONSHIP_HOH)
+            MONTHS_FED_TIME_LIMIT = (
+                instance["MONTHS_FED_TIME_LIMIT"]
+                if type(instance) is dict
+                else getattr(instance, "MONTHS_FED_TIME_LIMIT")
+            )
+            if FAMILY_AFFILIATION == 1 and (RELATIONSHIP_HOH == 1 or RELATIONSHIP_HOH == 2):
+                if MONTHS_FED_TIME_LIMIT is None or int(MONTHS_FED_TIME_LIMIT) < 1:
+                    return false_case
+                else:
+                    return true_case
+            else:
+                return true_case
+        except Exception:
```

Comment:
suggest adding debug log. Try to add log which includes the fields at least for more readable log